app/lib/test_applications.rb

```diff
@@ -132,34 +135,44 @@ def create_application_to_courses(
     end
 
     Audited.audit_class.as_user(candidate) do
-      traits = [%i[with_safeguarding_issues_disclosed
-                   with_safeguarding_issues_never_asked
-                   minimum_info]].sample
+      traits = if duplicate_candidates
+                 [%i[with_safeguarding_issues_disclosed
+                     with_safeguarding_issues_never_asked
+                     minimum_info duplicate_candidates]].sample
+               else
+                 [%i[with_safeguarding_issues_disclosed
+                     with_safeguarding_issues_never_asked
+                     minimum_info]].sample
+               end
       traits << :with_equality_and_diversity_data
 
+      first_name_on_application_form = duplicate_candidates ? 'John' : first_name
+      last_name_on_application_form = duplicate_candidates ? 'Smith' : last_name
```

Comment:
Maybe we could just do `ApplicationForm.last&.last_name || last_name` here so that we don't just have a single hard-coded duplicate name.